components/Search/SearchFound.js

```diff
@@ -3,39 +3,43 @@ import { useState } from "react";
 import FoundItem from "../FoundItem";
 import styled from "styled-components";
 import { useStore } from "../../store/useStore";
+import BurgerMenu from "../BurgerMenu";
 
 export default function SearchFound() {
   const [searchTerm2, setSearchTerm2] = useState("");
   const foundItems = useStore((state) => state.foundItems);
   return (
-    <Main>
-      <StyledHeadline>Found Items</StyledHeadline>
+    <>
+      <BurgerMenu />
```

Comment:
Same as above, there is no burger menu!? 😉

Reply:
Marc you are in any case very accurate :D